.gitlab-ci.yml

```diff
@@ -318,6 +319,35 @@ publish-adder-collator-image:
       # this artifact is used in trigger-simnet job
       dotenv: ./artifacts/collator.env
 
+.update_weights:                   &update-weights
+  stage:                           test
+  when:                            manual
+  tags:
+    - weights
+  variables:
+    RUNTIME:                       polkadot
+    # sccache causing issues...
```

Comment:
what's wrong with it? Is it (and redis) installed to those hosts?

Reply:
yeah I think it was due to redis. Should be resolved so I can remove this line now